src/h2load.cc

```diff
@@ -245,6 +293,18 @@ int Client::connect() {
   return 0;
 }
 
+void Client::timeout() {
+  process_timedout_streams();
+
+  disconnect();
+}
+
+void Client::restart_timeout() {
```

Comment:
Maybe this should be named to restart_inactivity_timeout() as it is ambiguous right now?